lib/deprecations.js

```diff
@@ -3,8 +3,6 @@
 const warning = require('fastify-warning')()
 module.exports = warning
 
-const warnName = 'PinoWarning'
+// const warnName = 'PinoWarning'
 
-warning.create(warnName, 'PINODEP008', 'prettyPrint is deprecated, look at https://github.com/pinojs/pino-pretty for alternatives.')
-
-warning.create(warnName, 'PINODEP009', 'The use of pino.final is discouraged in Node.js v14+ and not required. It will be removed in the next major version')
+// warning.create(warnName, 'PINODEP010', 'A new deprecation')
```

Comment:
stray comment?

Reply:
This is on purpose to avoid deleting this file.

Reply:
is the file needed? if not we should remove it - if we ever need it again it's in the commit history right?

Reply:
It's easier to keep it and update it as necessary.